envergo/hedges/services.py

```diff
@@ -65,6 +67,32 @@ def _check_plantation_result_matrix():
 _check_plantation_result_matrix()
 
 
+# This method is outside the PlantationEvaluator class because it makes it
+# easier to patch it in tests.
+def get_replantation_coefficient(moulinette):
+    """Get the "R" value.
+
+    It depends on the activated criteria.
+    """
+    ep_R = D("0")
+    if moulinette.ep.is_activated():
+        ep = moulinette.ep.criteria.all().first()
```

Comment:
```suggestion
        ep = moulinette.ep.criteria.first()
```
Je ne pense pas que ca change la query finale, mais il me semble que c'est suffisant.